google/cloud/completion_queue_test.cc

```diff
@@ -75,6 +75,36 @@ class MockOperation : public internal::AsyncGrpcOperation {
   MOCK_METHOD1(Notify, bool(bool));
 };
 
+/// @test A regression test for #5141
+TEST(CompletionQueueTest, TimerCancel) {
+  // There are a lot of magical numbers in this test, on my workstation I was
+  // able to reproduce #5141 99 out of 100 times on my workstation.
```

Comment:
"... on my workstation ... on my workstation" repetition?

Reply:
Fixed.

Reply:
Still sounds a little strange, but I'll leave it to you to decide.